### PR TITLE
Adding a active connections to summary.

### PR DESCRIPTION
**Pull request by crisnamurti**
Ok, identations removed.

### CODE REVIEW

File: pg_activity [hunk 1]
```diff
@@ -317,6 +321,7 @@ server activity monitoring.")
                 'indent': indent,
                 'io': delta_disk_io,
                 'tps': db_info['tps'],
+				'active_connections': active_connections,
```
Comment (julmon) [inline pg_activity]: Please reduce indentation

File: pg_activity [hunk 2]
```diff
@@ -334,6 +339,7 @@ server activity monitoring.")
                 indent,
                 delta_disk_io,
                 db_info['tps'],
+				active_connections,
```
Comment (julmon) [inline pg_activity]: Indentation please.

File: pgactivity/UI.py [hunk 3]
```diff
@@ -753,6 +753,7 @@ def __pause(self,):
                         self.uibuffer['indent'],
                         self.uibuffer['io'],
                         self.uibuffer['tps'],
+						self.uibuffer['active_connections'],
```
Comment (julmon) [inline pgactivity/UI.py]: Indentation please

File: pgactivity/UI.py [hunk 4]
```diff
@@ -1230,6 +1231,7 @@ def __poll_activities(self, interval, flag, indent, process = None, \
                     self.uibuffer['indent'],
                     self.uibuffer['io'],
                     self.uibuffer['tps'],
+					self.uibuffer['active_connections'],
```
Comment (julmon) [inline pgactivity/UI.py]: Indentation please.

File: pgactivity/UI.py [hunk 5]
```diff
@@ -1488,6 +1490,7 @@ def __poll_waiting_blocking(self, interval, flag, indent, \
                 self.uibuffer['indent'],
                 self.uibuffer['io'],
                 self.uibuffer['tps'],
+				self.uibuffer['active_connections'],
```
Comment (julmon) [inline pgactivity/UI.py]: Indentation please.

File: pgactivity/UI.py [hunk 6]
```diff
@@ -1879,6 +1891,7 @@ def refresh_window(self, procs, extras, flag, indent, ios, \
             dbname,
             ios,
             tps,
+			active_connections,
```
Comment (julmon) [inline pgactivity/UI.py]: Indentation please

File: pg_activity [hunk 7]
```diff
@@ -275,6 +275,8 @@ server activity monitoring.")
         delta_disk_io = None
         # get DB informations
         db_info = PGAUI.data.pg_get_db_info(None, using_rds = options.rds)
+        # get active connections
+        active_connections = PGAUI.data.pg_get_active_connections()
```
Comment (julmon) [inline pg_activity]: At this point, we don't need to get active connections count, you can remove those 2 lines. Thanks.

File: pg_activity [hunk 8]
```diff
@@ -303,6 +305,8 @@ server activity monitoring.")
             # refresh the winodw
             db_info = PGAUI.data.pg_get_db_info(db_info, using_rds = options.rds)
             PGAUI.set_max_db_length(db_info['max_length'])
+            # get active connections
+            active_connections = PGAUI.data.pg_get_active_connections()
```
Comment (julmon) [inline pg_activity]: Please remove the 4 trailing spaces.

**Comment by julmon** [comment]
Actually, active connections counter doesn't seem to be accurate because we run SQL queries into their own transaction. I'm going to fix that once your PR is merged.

Thank you,

File: pg_activity (hunk 8 above)
Comment (crisnamurti) [reply pg_activity]: Which editor do you use for python?
If vim can you send me the vimrc?
Thanks.

Comment (julmon) [reply pg_activity]: Here it is : https://gist.github.com/julmon/674751b50a42d42db54fde99ad782f25

Comment (crisnamurti) [reply pg_activity]: Thanks. Trailing spaces removed.

**Pull request merged**